config/internal/apiserver/default/deployment.yaml.tmpl

```diff
@@ -120,10 +120,10 @@ spec:
             - name: SIGNED_URL_EXPIRY_TIME_SECONDS
               value: "{{.APIServer.ArtifactSignedURLExpirySeconds}}"
             {{ if .PodToPodTLS }}
-            - name: ML_PIPELINE_TLS_ENABLED
-              value: "true"
             - name: METADATA_TLS_ENABLED
               value: "true"
+            - name: CABUNDLE_SECRET_NAME
+              value: "proxy-tls"
```

Comment:
@alyssacgoins Did you mean to use the secret name `ds-pipelines-proxy-tls-{{.Name}}` ?